monitoring/opencensus/.python-version

```diff
@@ -0,0 +1 @@
+3.6.12
```

Comment:
Is this file consumed by open census?

Reply:
No, it's created by `pyenv`. Going to delete it.